__tests__/api.test.js

```diff
@@ -63,4 +63,32 @@ describe("API endpoints", () => {
                 expect(article).toEqual(expected)
         });
     })
+    test("/api/articles endpoint should return all articles in an array", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+            .then((result) => {
+                const articles = result.body.articles;
+                const expected = 12
+                expect(articles.length).toBe(expected)
```

Comment:
Good assertions here Chetin, well done. Good to check the length, and checking that structure of the data (the shape) is as you expect it to be.